go/libkb/gpg_cli.go

```diff
@@ -170,6 +170,35 @@ func (g *GpgCLI) ExportKey(k PGPKeyBundle) (err error) {
 	return PickFirstError(e1, e2, e3)
 }
 
+func (g *GpgCLI) Sign(fp PGPFingerprint, payload []byte) (string, error) {
+	arg := RunGpg2Arg{
+		Arguments: []string{"--armor", "--sign", "--default-key", fp.String()},
```

Comment:
Why --default-key? I guess check that is correct. I thought -u is what is needed.